# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../meson/no-lzma-dep.patch
# Copyright (C) 2020 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- meson-0.52.1/mesonbuild/mdist.py.vanilla	2020-01-22 16:02:40.876499986 +0100
+++ meson-0.52.1/mesonbuild/mdist.py	2020-01-22 16:03:11.444500001 +0100
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 
-import lzma
+#import lzma
 import gzip
 import os
 import sys
